test/development/duplicate-pages/duplicate-pages.test.ts

```diff
@@ -0,0 +1,14 @@
+import { createNextDescribe } from 'e2e-utils'
+
+createNextDescribe(
+  'Handles Duplicate Pages',
+  {
+    files: __dirname,
+  },
+  ({ next }) => {
+    it('Shows warning in development', async () => {
+      await next.render('/hello')
+      expect(next.cliOutput).toMatch(/Duplicate page detected/)
```

Comment:
Can we check if it's only logged once? If it can be split by a delimiter into 2 parts, then it only contains 1 delimiter 

```suggestion
      expect(next.cliOutput.split('Duplicate page detected').length).toBe(2)
```

Reply:
It will be logged multiple times currently so that wouldn't work exactly.